C++/source/number_theory/Median_In_a_Stream.cpp

```diff
@@ -0,0 +1,77 @@
+/*
+    Median in a stream of integers(running integers)
+    ------------------------------------------------
+    An algorithm to find the median of elements read so far from
+    a data stream in a efficient way.
+
+    Assumptions: There are no duplicates.
+
+    Time Complexity
+    ---------------
+    O(NlogN), where N is the number of elements.
+
+    Space Complexity
+    ----------------
+    O(N^2)
+
+    Auxilliary Space
+    ----------------
+    O(N)
+
+*/
+//Explanation link :https://www.youtube.com/watch?v=VmogG01IjYc
+#include<iostream>
+#include<algorithm>
+#include<set>
+#include<math.h>
+#include<iterator>
+#include<queue>
+
+using namespace std;
+
+typedef long long ll;
+priority_queue <ll> great;
+priority_queue<ll,vector <ll>,greater<ll> > shorter;
```

Comment:
Make these queues local to `main()`

Reply:
These queues should be declared inside `main()` and passed to whichever function needs them by reference.